templates/admin/group/group_property_editor.html.ep

```diff
@@ -26,15 +26,45 @@
-            % if (!$is_parent) {
-                        <input type="number" min="1" class="form-control" id="editor-size-limit" name="size_limit_gb" value="<%= $group->size_limit_gb %>"> GiB
-            % }
-            % else {
-                        <input type="number" min="1" class="form-control" id="editor-size-limit" name="default_size_limit_gb" value="<%= $group->default_size_limit_gb %>"> GiB
-            % }
+                        <div class="container">
+                            <div class="row">
+                                <div class="col-sm">
+                                    <input id="editor-size-limit" name="size_limit_gb" type="number" min="1" class="form-control"
+                                        value="<%= $group->get_column('size_limit_gb') // '' %>"
+                                        placeholder="<%= $is_parent
+                                                ? 'none, limits set on job group level apply'
+                                                : 'default, configured to ' . app->config->{default_group_limits}->{asset_size_limit} %>">&nbsp;GiB
+                                </div>
+                                <div class="col-md-auto">
+                                    <button id="clear-size-limit-button"
+                                            class="btn btn-outline-info btn-sm" onclick="$('#editor-size-limit').val('');"
+                                            data-toggle="tooltip" data-placement="bottom"
+                                            title="<%= $is_parent ? 'to use the limits set on job group level instead'
+                                                                  : 'to use configured default instead' %>">
+                                        <i class="fas fa-eraser"></i> Clear limit
+                                    </button>
+                                </div>
+                            </div>
+                        </div>
                     </div>
                 </div>
-            % if (!$is_parent) {
+            % my ($overridden_asset_limit, $parent_group);
+            % $overridden_asset_limit = $parent_group->size_limit_gb if !$is_parent && defined ($parent_group = $group->parent);
+            % if (defined $overridden_asset_limit) {
+                <div class="form-group row">
+                    <label for="editor-size-limit-warning" class="col-sm-2 control-label"><i class="fas fa-exclamation-triangle"></i></label>
+                    <div class="col-sm-10">
+                        <span id="editor-size-limit-warning">
+                            <div>
+                                The size limit for this group is overriden by its <a href="<%= url_for 'admin_edit_parent_group', groupid => $parent_group->id %>">parent job
+                                group</a> to <%= OpenQA::Utils::human_readable_size($overridden_asset_limit * 1024 * 1024 * 1024) %>. Therefore
```

Comment:
You can import functions into templates (`% use OpenQA::Utils 'human_readable_size';`), no need to always use the full class name.